### PR TITLE
debug CONFIG_FILE_PATH value

### PR DESCRIPTION
**Pull request by mboukhalfa**
debug

### CODE REVIEW

File: test/e2e/pivoting_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func pivoting() {
 	}
 	_, err := targetClusterClientSet.CoreV1().Namespaces().Create(ctx, ironicNamespace, metav1.CreateOptions{})
 	Expect(err).To(BeNil(), "Unable to create the Ironic namespace")
-
+	Logf("os.Getenv: %v", os.Getenv("CONFIG_FILE_PATH"))
```
Comment (furkatgofurov7) [inline test/e2e/pivoting_test.go:49]: I do not think this is the right way to access this var when using Logf, can we try, i.e:
```suggestion
        var (
		configFilePath = e2eConfig.GetVariable("CONFIG_FILE_PATH")
	)
	Logf("config file path is: %v", configFilePath)
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: test/e2e/pivoting_test.go (hunk 1 above)
Comment (mboukhalfa) [reply test/e2e/pivoting_test.go:49]: Should I use  e2eConfig.GetVariable("CONFIG_FILE_PATH") also in : 
```
By("Initialize Provider component in target cluster")
	clusterctl.Init(ctx, clusterctl.InitInput{
		KubeconfigPath: targetCluster.GetKubeconfigPath(),
		// TODO (furkat), to uplift CAPI to v1.1.0-beta.1 pre-release
		// in pivoting tests we are pointing to overrides folder as ClusterctlConfigPath.
		// We have to revert this to point to e2e ClusterctlConfigPath once there is an
		// actual latest release with v1.1.X tag.
	ClusterctlConfigPath:    e2eConfig.GetVariable("CONFIG_FILE_PATH")
```
To make sure it has the same value printed?

Comment (furkatgofurov7) [reply test/e2e/pivoting_test.go:49]: Sure, go ahead

**Pull request closed without merging**